src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -295,6 +295,11 @@ func (server *Server) checkForBannedCountry(req *http.Request) error {
 }
 
 func (server *Server) lookupCountry(req *http.Request) (string, error) {
+	// Use the country CloudFlare gives us if it's available.
+	cf := req.Header.Get("Cf-Ipcountry")
```

Comment:
> this could be done independently of the country/ip lookup above, right?

It's actually always done independently -- i.e. this code is only currently run on flashlight servers. As far as I know there's only one standard for two letter country codes, so I think we're fine there. The prior lookup, however, also returns codes that are ultimately from the "Cf-Ipcountry" header (i.e. they're returned from Flashlight servers that are using that header), so there's no way for them to conflict.